pom.xml

```diff
@@ -121,7 +121,7 @@
     <nuxeo.webengine.version>8.2-SNAPSHOT</nuxeo.webengine.version>
     <nuxeo.features.version>8.2-SNAPSHOT</nuxeo.features.version>
 
-    <nuxeo.connect.client.version>1.4.24</nuxeo.connect.client.version>
+    <nuxeo.connect.client.version>1.4.25-SNAPSHOT</nuxeo.connect.client.version>
```

Comment:
missing exclude in maven-enforcer-plugin in nighlty profile, you will break the nightly build